File: code/stacy/javascript/Lecture/Vanilla_js/first_js/grading.py


number_grade = int(input("Enter your grade \n\t> "))

grade_table = {
    "10": "A+",
    "9": "A",
    "8": "B",
    "7": "C",
    "6": "D",
}

key = number_grade // 10

letter_grade = grade_table.get(str(key), "F")

print(f"You got a(n) {letter_grade}")

File: code/stacy/javascript/Lecture/Vanilla_js/first_js/grading.js


const numberGrade = Number(prompt("Enter your grade \n\t> "))

const gradeTable = {
    "10": "A+",
    "9": "A",
    "8": "B",
    "7": "C",
    "6": "D"
}

const key = Math.floor(numberGrade / 10)

let letterGrade = gradeTable[String(key)]

if (letterGrade == undefined){letterGrade = "F"}

console.log(`You got a(n) ${letterGrade}`)